wolfcrypt/test/test.c

```diff
@@ -22837,6 +22837,27 @@ static int myCryptoDevCb(int devIdArg, wc_CryptoInfo* info, void* ctx)
         info->rng.rng->devId = devIdArg;
     #endif
     }
+    else if (info->algo_type == WC_ALGO_TYPE_SEED) {
+    #ifndef WC_NO_RNG
+        static byte seed[] = { 0x00, 0x00, 0x00, 0x01 };
+        word32 len;
+        int i;
+
+        /* wc_GenerateSeed is a local symbol so we need to fake the entropy. */
+        while (info->seed.sz > 0) {
+            len = sizeof(seed);
```

Comment:
You may get a cast warning here between size_t and word32.

Reply:
explicit cast now